components/server/src/api/prebuild-service-api.ts

```diff
@@ -124,6 +124,12 @@ export class PrebuildServiceAPI implements ServiceImpl<typeof PrebuildServiceInt
         const it = this.prebuildManager.getAndWatchPrebuildStatus(ctxUserId(), filter, { signal: ctxSignal() });
 
         for await (const pb of it) {
+            if (ctxSignal().aborted) {
```

Comment:
I thought they will be cancelled together here https://github.com/gitpod-io/gitpod/blob/097e2c3c29b085ce99ecc34c4b4a7d5fc2e0a1d0/components/gitpod-protocol/src/generate-async-generator.ts#L27

But not sure if it's actually working

Reply:
Should work? because we have test cases for  this function

Reply:
@geropl  see also https://github.com/connectrpc/connect-es/blob/v1.1.2/packages/connect-node/src/node-universal-handler.ts#L227-L241

Reply:
> Should we try to catch that error and silence it? WDYT? 🤔

👍